dask/dataframe/core.py

```diff
@@ -736,14 +736,14 @@ def map_partitions(self, func, *args, **kwargs):
         Parameters
         ----------
         func : function
-            Function applied to each partition.
+            Map function applied to each partition. If this function accepts the special ``partition_info`` keyword
+            argument, it will recieve information on the partition's relative location within the dataframe.
         args, kwargs :
-            Arguments and keywords to pass to the function. The partition will
-            be the first argument, and these will be passed *after*. Arguments
-            and keywords may contain ``Scalar``, ``Delayed``, ``partition_info``
-            or regular python objects. DataFrame-like args (both dask and
-            pandas) will be repartitioned to align  (if necessary) before
-            applying the function (see ``align_dataframes`` to control).
+            Positional and keyword arguments to pass to the function. Positional arguments are computed on a
+            per-partition basis, while keyword arguments are shared across all partitions. The partition itself will be
+            the first positional argument, with all other arguments passed *after*. Arguments can be ``Scalar``,
+            ``Delayed``, or regular Python objects. DataFrame-like args (both dask and pandas) will be repartitioned to
+            align (if necessary) before applying the function; see ``align_dataframes`` to control this behavior.
```

Comment:
Our docstrings are generally line wrapped to ~80 characters, would you mind fixing these to match that convention?